src/gsi/gsimod.F90

```diff
@@ -1985,6 +1987,12 @@ subroutine gsimain_initialize
   regional=wrf_nmm_regional.or.wrf_mass_regional.or.twodvar_regional.or.nems_nmmb_regional .or. cmaq_regional
   regional=regional.or.fv3_regional.or.fv3_cmaq_regional
 
+! Force turn off MGBF-based localization except for regional application
+  if(.not.regional.and.l_mgbf_loc) then
+     l_mgbf_loc=.false.
+     if (mype==0) write(6,*)'GSIMOD:  ***WARNING*** reset l_mgbf_loc=',l_mgbf_loc
```

Comment:
Please add comments like "for global app, l_mgbf_loc is not applicable, reset ..."

Reply:
Corrected.